locales/de/subtitles/giacomina-castagnetti/giacomina-castagnetti-01.yml

```diff
@@ -179,30 +179,31 @@ de:
           s: 33
           from: 00:03:42,08
           to: 00:03:52,29
-          text: We didn’t take much math, but we spent a lot of time doing gymnastics,
-            as we had to forge our physique.
+          text: Wir hatten nicht viel Mathe, aber wir verbrachten viel Zeit mit Gymnastik
+            weil wir unsere Körper ertüchtigen sollten.
         sequence-34:
           s: 34
           from: 00:03:52,29
           to: 00:04:03,05
-          text: Since we didn’t have a gym, every day we spent two hours in the school
-            courtyard doing physical education,
+          text: Weil wir keine Turnhalle hatten haben wir jeden Tag zwei Stunden im
+            Hof Sport getrieben,
         sequence-35:
           s: 35
           from: 00:04:03,05
           to: 00:04:06,20
-          text: even in the winter when it turned cold.
+          text: selbst im Winter, wenn es kalt war.
         sequence-36:
           s: 36
           from: 00:04:06,20
           to: 00:04:16,07
-          text: We were all excited for the gymnastics display we were working on.
+          text: Wir waren alle wegen der Gymnastikaufführung aufgeregt,
```

Comment:
How about `Wir waren alle aufgeregt wegen der Gymnastikaufführung,...`. Sounds better for me and it avoids the **auf**führung **auf**geregt.